at_secondary/at_persistence_secondary_server/pubspec.yaml

```diff
@@ -19,22 +19,23 @@ dependencies:
   at_utf7: ^1.0.0
   meta: ^1.7.0
 
-# dependency_overrides:
-#   at_persistence_spec:
-#     git:
-#       url: https://github.com/atsign-foundation/at_server.git
-#       path: at_persistence/at_persistence_spec
-#       ref: trunk
+dependency_overrides:
+  at_persistence_spec:
+    git:
+      url: https://github.com/atsign-foundation/at_server.git
+      path: at_persistence/at_persistence_spec
+      ref: at_persistence_encode_public_data
+  at_commons:
+    git:
+      url: https://github.com/atsign-foundation/at_tools.git
+      path: at_commons
+      ref: encode_public_data
```

Comment:
at_commons is published with the dependent changes for this PR?

Reply:
@murali-shris : Not yet, waiting to publish all the packages at once.